app/models/concerns/foreman_openscap/policy_extensions.rb

```diff
@@ -244,5 +251,28 @@ def populate_overrides(puppetclass, hostgroup)
         end
       end
     end
+
+    def set_certificates_overrides(puppetclass)
+      [CA_FILE_PARAMETER, HOST_CERTIFICATE_PARAMETER, HOST_PRIVATE_KEY_PARAMETER].each do |override_parameter|
+        unless certificate_param = puppetclass.class_params.where(:key => override_parameter).first
+          errors[:base] << _("Puppet class %{class} does not have %{parameter} class parameter.") % {:class => SCAP_PUPPET_CLASS, :parameter => override_parameter}
+          return false
+        end
+        certificate_param.override = true
+        case override_parameter
+          when CA_FILE_PARAMETER
+            certificate_param.default_value = '/etc/rhsm/ca/katello-server-ca.pem'
+          when HOST_CERTIFICATE_PARAMETER
+            certificate_param.default_value = '/etc/pki/consumer/cert.pem'
+          when HOST_PRIVATE_KEY_PARAMETER
+            certificate_param.default_value = '/etc/pki/consumer/key.pem'
+        end
```

Comment:
Oh, can you not read the rhsm.conf on the client uploading tool instead of sending them via Puppet?

Depending on when the system was registered these could be different.... at one point, the file wasn't named /etc/rhsm/ca/katello-server-ca.pem.